orc8r/cloud/helm/orc8r/charts/logging/values.yaml

```diff
@@ -27,6 +27,9 @@ fluentd_daemon:
 fluentd_forward:
   create: false
 
+  # Domain-proxy output
+  dp_output: true
```

Comment:
If we enable this by default, will we get error if domain proxy helm charts are not deployed?

Reply:
Will not. However, we can disable it by default.

Reply:
Then it's fine.